features/admin.actions.v1/pages/action-configuration-page.tsx

```diff
@@ -0,0 +1,428 @@
+                            }
+                            showEmptyLink={ false }
+                        >
+                            { t("common:learnMore") }
+                        </DocumentationLink>
+                    </>
+                );
+            case ActionsConstants.ACTION_TYPES.PRE_UPDATE_PROFILE.getApiPath():
+                return (
+                    <>
+                        { t("console:manage.features.actions.types.preUpdateProfile.description.expanded") }
+                        <DocumentationLink
+                            link={
+                                getLink("develop.actions.types.preUpdateProfile.learnMore")
+                            }
+                            showEmptyLink={ false }
+                        >
+                            { t("common:learnMore") }
+                        </DocumentationLink>
+                    </>
+                );
+            case ActionsConstants.ACTION_TYPES.PRE_REGISTRATION.getApiPath():
+                return (
+                    <>
+                        { t("console:manage.features.actions.types.preRegistration.description.expanded") }
+                        <DocumentationLink
+                            link={
+                                getLink("develop.actions.types.preRegistration.learnMore")
+                            }
+                            showEmptyLink={ false }
+                        >
+                            { t("common:learnMore") }
+                        </DocumentationLink>
+                    </>
+                );
+            default:
+                return "";
+        }
+    };
+
+    /**
+     * This renders the toggle button for action status.
+     */
+    const actionToggle = (): ReactElement => {
+
+        const handleToggle = (e: SyntheticEvent, data: CheckboxProps) => {
+            setIsActive(data.checked);
+
+            if (data.checked) {
+
+                changeActionStatus(actionTypeApiPath, actionInitialValues.id, ActionsConstants.ACTIVATE)
+                    .then(() => {
+                        handleSuccess(ActionsConstants.UPDATE);
+                    })
+                    .catch((error: AxiosError) => {
+                        handleError(error, ActionsConstants.UPDATE);
+                    })
+                    .finally(() => {
+                        mutateActions();
+                    });
+            } else {
+                changeActionStatus(actionTypeApiPath, actionInitialValues.id, ActionsConstants.DEACTIVATE)
+                    .then(() => {
+                        handleSuccess(ActionsConstants.UPDATE);
+                    })
+                    .catch((error: AxiosError) => {
+                        handleError(error, ActionsConstants.UPDATE);
+                    })
+                    .finally(() => {
+                        mutateActions();
+                    });
+            }
+        };
```

Comment:
Reduce duplicate code here.
```suggestion
            if (data.checked) {
                changeActionStatus(
                    actionTypeApiPath,
                    actionInitialValues.id,
                    data.checked ? ActionsConstants.ACTIVATE : ActionsConstants.DEACTIVATE
                )
                    .then(() => {
                        handleSuccess(ActionsConstants.UPDATE);
                    })
                    .catch((error: AxiosError) => {
                        handleError(error, ActionsConstants.UPDATE);
                    })
                    .finally(() => {
                        mutateActions();
                    });
            }
        };
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6722/commits/5e3564e0797c09f4f20b71837b928bc9fa57c847